TheWarWithin/Priorities/RogueAssassination.simc

```diff
@@ -7,29 +7,31 @@ actions.precombat+=/variable,name=trinket_sync_slot,value=2,if=trinket.2.has_sta
 actions.precombat+=/variable,name=effective_spend_cp,value=cp_max_spend-2<?5*talent.hand_of_fate
 # Pre-cast Slice and Dice if possible
 actions.precombat+=/stealth
-actions.precombat+=/slice_and_dice,precombat_seconds=1,if=refreshable&!buff.indiscriminate_carnage.up
+actions.precombat+=/slice_and_dice
```

Comment:
We need to keep these conditions so we don't try to SnD moving pack to pack when SnD is already up.

Reply:
Oh ok